### PR TITLE
[Minor] Fix padding on home page

### PR DESCRIPTION
**Pull request by prabhjyotsingh**
This is a minor CSS to add padding on the table and paginator. It does not make or break anything but makes this page looks slightly different from others. 
I've added before/after below for reviewers. 

Let me know if this requires any more changes.

Before:
before

After:
option2

### CODE REVIEW

**Comment by prabhjyotsingh** [comment]
CI failed for "Quarantined tests" looks unrelated.

**Review by ryanahamilton — CHANGES_REQUESTED** [review]
The table intentionally does not have extra padding given its extra-wide contents. The actual issue here is that the pagination buttons and "Showing 1-2…" label aren't supposed to be within the gray box, they are supposed to be below/outside of it. Their current positions actually appear correct, they just shouldn't have the gray background. Want to take a stab at resolving?

**Comment by prabhjyotsingh** [comment]
@ryanahamilton  Sure, which one looks better?

Option-1:
option1

Option-2:
option2

**Review by bbovenzi — APPROVED** [review]
Looking good now. Great work!

File: airflow/www/static/css/dags.css [hunk 1]
```diff
@@ -31,10 +31,12 @@
 .dags-table-body {
   margin: 0 1px;
   overflow-x: auto;
+  padding-bottom: 34px;
+}
 
-  /* This allows small dropdowns to overlap the table element */
-  padding-bottom: 55px;
-  margin-bottom: -55px;
+.no-padding {
```
Comment (ryanahamilton) [inline airflow/www/static/css/dags.css]: This class name suggests `padding: 0;` but it is only setting x-axis.
```suggestion
.no-x-padding {
```

File: airflow/www/templates/airflow/dags.html [hunk 2]
```diff
@@ -141,7 +141,7 @@ <h2>{{ page_title }}</h2>
           </thead>
           <tbody>
             {% if dags|length == 0 %}
-              <tr><td colspan="9">No results</td></tr>
+              <tr><td colspan="10">No results</td></tr>
```
Comment (ryanahamilton) [inline airflow/www/templates/airflow/dags.html:144]: Good catch!

**Review by ryanahamilton — CHANGES_REQUESTED** [review]
Just a small nit regarding class naming, but otherwise LGTM.

**Review by ryanahamilton — APPROVED** [review]
Nice work. Always fun how these seemingly simple changes open a whole can of worms!

**Pull request merged**